pulsar-client-admin-api/src/main/java/org/apache/pulsar/client/admin/Brokers.java

```diff
@@ -32,6 +32,41 @@
  * Admin interface for brokers management.
  */
 public interface Brokers {
+    /**
+     * Get the list of active brokers in the local cluster.
+     * <p/>
+     * Get the list of active brokers (web service addresses) in the local cluster.
+     * <p/>
+     * Response Example:
+     *
+     * <pre>
+     * <code>["prod1-broker1.messaging.use.example.com:8080", "prod1-broker2.messaging.use.example.com:8080"
+     * * * "prod1-broker3.messaging.use.example.com:8080"]</code>
+     * </pre>
+     *
+     * @return a list of (host:port)
+     * @throws NotAuthorizedException
+     *             You don't have admin permission to get the list of active brokers in the cluster
+     * @throws PulsarAdminException
+     *             Unexpected error
+     */
+    List<String> getActiveBrokers() throws PulsarAdminException;
```

Comment:
I think we can continue to use `getActiveBrokers(String cluster)`.

When the `cluster` is `null`, you can request  `/`, otherwise, you can request `/{cluster}`.

Reply:
User coud also use `getActiveBrokers(null)` to get broker list for local cluster, but it's a littile strange, so I add `getActiveBrokers()`

Reply:
Yes right. But seems no need to modify other part.